app/menu.js

```diff
@@ -393,18 +392,17 @@ const init = (settingsState, args) => {
           accelerator: 'Shift+CmdOrCtrl+Delete',
           enabled: siteUtil.hasNoTagSites(appStore.getState().get('sites')),
```

Comment:
Actually I'll just simplify this to not disable the item, there is no need to disable the functionality even if it does no-op because already cleared.  And disabling adds confusion for why it is disabled.